king_phisher/client/windows/plugin_manager.py

```diff
@@ -310,11 +311,21 @@ def _add_catalog_to_tree_tsafe(self, catalog):
 					version=plugin_info['version'],
 					visible_enabled=True,
 					visible_installed=True,
-					sensitive_installed=True,
+					sensitive_installed=self._is_sensitive_installed(self.catalog_plugins.compatibility(catalog.id, repo.id, plugin_name)),
 					type=_ROW_TYPE_PLUGIN
 				))
 		gui_utilities.glib_idle_add_once(self.__store_add_node, catalog_node)
 
+	def _is_sensitive_installed(self, compatibility_details):
+		for detail in compatibility_details:
+			if detail.type.name == 'platforms' and not detail.comptabile:
```

Comment:
Compare `detail.type` to the actual `RequirementType` enum after making it public.